posthog/client.py

```diff
@@ -148,7 +148,7 @@ def sync_execute(
     flush=True,
     client_query_id: Optional[str] = None,
     client_query_team_id: Optional[int] = None,
-):
+) -> Optional[Any]:
```

Comment:
Is the expectation now to explicitly cast the results we get from sync_execute, wherever sync_execute is called?

Reply:
Yes, as the `clickhouse-driver` doesn't support types we've added this default.